<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PK Übungstool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Gruppenauswahl:</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_repo_path">
           <property name="text">
            <string>Pfad zum PK-Repo:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_edit_repo_path"/>
         </item>
         <item>
          <widget class="QPushButton" name="button_select_repo_path">
           <property name="text">
            <string>Auswählen</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_group_type">
           <property name="text">
            <string>Gruppentype:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="group_type_combobox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_day">
           <property name="text">
            <string>Übungstag:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="group_day_combobox"/>
         </item>
         <item>
          <widget class="QLabel" name="label_group">
           <property name="text">
            <string>Gruppe:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="group_combobox">
           <property name="focusPolicy">
            <enum>Qt::WheelFocus</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Gruppen-Info:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QLabel" name="label_tutor1_name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_instructor">
         <property name="text">
          <string>Übungsleiter:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_instructor_name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_tutor1">
         <property name="text">
          <string>Tutor 1:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_tutor2_name">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_tutor2">
         <property name="text">
          <string>Tutor 2:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QTableWidget" name="table_widget">
      <property name="focusPolicy">
       <enum>Qt::WheelFocus</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLineEdit" name="console"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Befehl: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="console_output">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_new"/>
   <addaction name="action_export"/>
   <addaction name="action_undo"/>
   <addaction name="action_redo"/>
   <addaction name="action_add_student"/>
  </widget>
  <action name="action_export">
   <property name="text">
    <string>Exportieren</string>
   </property>
  </action>
  <action name="action_new">
   <property name="text">
    <string>Neu</string>
   </property>
   <property name="toolTip">
    <string>Neu</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="text">
    <string>Zurück</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="text">
    <string>Vor</string>
   </property>
  </action>
  <action name="action_add_student">
   <property name="text">
    <string>Student hinzufügen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
